plugins/modules/ec2_vol.py

```diff
@@ -408,14 +423,23 @@ def update_volume(module, ec2_conn, volume):
                 throughput_changed = True
                 req_obj['Throughput'] = target_throughput
 
-        changed = iops_changed or size_changed or type_changed or throughput_changed
+        target_multi_attach = module.params.get('multi_attach')
+        multi_attach_changed = False
+        if target_multi_attach is not None:
+            original_multi_attach = volume['multi_attach_enabled']
+            if target_multi_attach != original_multi_attach:
+                multi_attach_changed = True
+                req_obj['MultiAttachEnabled'] = target_multi_attach
```

Comment:
If `multi_attach` is not request it not requested, the return value will not response the current multi attach status, right?

Reply:
this part is when modifying the volume, if multi_attach is not requested the volume will remain as it was before the modification